src/content/docs/developer-tools/kinde-api/connect-to-kinde-api.mdx

```diff
@@ -0,0 +1,51 @@
+---
+page_id: 50284476-2442-414c-af20-01ed3ef4ca4e
+title: Set up Kinde Management API access
+sidebar:
+  order: 1
+relatedArticles:
+  - 51899f7f-3436-46e0-9a1b-6ecc3603a0df
+  - 6bf993fc-a195-4836-8eaf-133812be8876
+  - 601dd8c5-6ee1-474f-ad36-201e65280462
+app_context:
+  - m: settings
+    s: applications
+---
+
+Here's the process for setting up your access to the Kinde Management API.
```

Comment:
In order to call the Kinde management API you will need a Kinde management API access token. 

These tokens are generated by using an authorized M2M application to call the Kinde token endpoint.

This guide shows you how to set up a new M2M application that is authorized for Kinde management API access.
 
## Step 1: Add a machine to machine application

1. Go to **Settings > Applications**.
2. Select **Add Application**.
3. In the dialog that opens, give the application a name, and select **Machine to Machine** as the **Application type**.
4. Select **Save**. App keys - including Domain details, Client ID and Client Secret - are issued for the application. 
5. Copy the app keys. You need these details to test or initiate access requests from your application or service to Kinde's API. See Step 3 below.

## Step 2: Authorize the M2M application 

Only authorized M2M applications are allowed to access the Kinde management API. Without this step you will receive an error when calling the Kinde management API.

1. Select **View details** on the M2M application you just created. 
2. On the left, select **APIs**.
3. Select the three dots next to the Kinde management API, then choose **Authorize application**.

## Step 3: Select scopes 

Each Kinde endpoint requires a specific permission to be present in the token to grant access to it. These are known as scopes. If a scope is not present in the access token the API will return an error indicating which scope is required. 

To add scopes:

1. Select the three dots next to the Kinde management API, then choose **Manage scopes**. 
2. Select the scopes you want to include in the token. For maximum security only enable the minimum scopes you require. 
3. Select **Save**.

## Step 4: Get an access token and call the  management API

Follow this [guide for accessing Kinde Mangement API token](/developer-tools/kinde-api/access-token-for-api/). The guide includes steps for how to obtain and use a token.